CMakeLists.txt

```diff
@@ -1,6 +1,5 @@
 CMAKE_MINIMUM_REQUIRED(VERSION 3.4.0)
 INCLUDE(GNUInstallDirs)
-PROJECT(hiredis)
```

Comment:
Instead of removing this line I would like to have a
`PROJECT(hiredis LANGUAGES "C")`
before 
`INCLUDE(GNUInstallDirs)`
since `GNUInstallDirs` needs a language to be set:

```
CMake Warning (dev) at /snap/cmake/580/share/cmake-3.17/Modules/GNUInstallDirs.cmake:225 (message):
  Unable to determine default CMAKE_INSTALL_LIBDIR directory because no
  target architecture is known.  Please enable at least one language before
  including GNUInstallDirs.
Call Stack (most recent call first):
  CMakeLists.txt:2 (INCLUDE)
```

Reply:
Thanks for your review, I updated the PR to move `INCLUDE(GNUInstallDirs)` after the `PROJECT` statement.

Reply:
Ah yes, even better! The provided `CMAKE_INSTALL_` variables weren't needed until the install section a bit later..